core/templates/self_list.h

```diff
@@ -77,7 +77,10 @@ class SelfList {
 		}
 
 		void remove(SelfList<T> *p_elem) {
-			ERR_FAIL_COND(p_elem->_root != this);
+			if (p_elem->root != this) {
```

Comment:
I'd like someone to verify that this change is indeed required, we are losing information by removing the error here